### PR TITLE
updated pt_PT help translations

### PR DESCRIPTION
**Pull request by pmraps**

### CODE REVIEW

File: help/pt/Help_fdl-appendix.xml [hunk 1]
```diff
@@ -593,7 +590,7 @@
     <title>9. TERMINATION</title>
     <para>
       You may not copy, modify, sublicense, or distribute the <link
-      linkend="fdl-document">Document</link> except as expressly
+      linkend="fdl-document">Document</link> excepto as expressly
```
Comment (fellen) [inline help/pt/Help_fdl-appendix.xml]: The license is only valid in english. But you can link the original and add an unofficial translation. Search the fsf.org for accepted translations.

File: help/pt/Help_ch_GUIMenus.xml [hunk 2]
```diff
@@ -108,22 +105,22 @@
...
+            <para>Faz uma descrição do item de menu activo e mostra uma barra de 
             progresso</para>
           </listitem>
         </varlistentry>
       </variablelist>
 
 
       <note>
-        <para>O conteúdo e opções dos vários submenus das diferentes secções da 
+        <para>O conteúdo e opções dos vários sub-menus das diferentes secções da
```
Comment (fellen) [inline help/pt/Help_ch_GUIMenus.xml]: Was this intended?

File: help/pt/Help_ch_GUIMenus.xml [hunk 3]
```diff
@@ -229,10 +226,10 @@
               </row>
 
               <row>
-                <entry><para><menuchoice><guisubmenu>Importar</guisubmenu><guimenuitem/></menuchoice></para></entry>
+                <entry><para><menuchoice><guisub-menu>Importar</guisub-menu><guimenuitem/></menuchoice></para></entry>
```
Comment (fellen) [inline help/pt/Help_ch_GUIMenus.xml]: See my other comment.

**Comment by pmraps** [comment]
Hi,
Sure, no problem, when I finally learn git and github, I'll be happy to do all that stuff :)
Now, seriously, I'll try to correct some of those on the weekend, using something like Geany IDE, which shouldn't change indents and other things. And them I'll try to follow @gjanssens advise, by deleting my actual fork, fork maint branch again, copy-paste the changes and creating a PR. That is the best I can do.

But you developers should really consider start using a service like weblate for the translations.
That is adequate for non-developers like me. Not git or github, which are for professional programing people.

Anyway, many thanks for your patience and comments. I do appreciate them. If you want, this PR can be closed, since I don't know how to solve it here.

**Comment by jralls** [comment]
@pmraps, thanks for the pointer to [weblate](https://github.com/WeblateOrg/weblate), I'd never heard of it before.

**Comment by fellen** [comment]
Hello Pedro,
I had your PR totally forgotten.

Finally after some merging and fixing the docbook syntax I applied your PR:
1. <gui... elements> can not contain <application> elements. Fixed.
1. I enabled your translator copyright. :-)
1. Please review the "sub-" strings, they seem mostly an unintended change to me.
1. Did you change Entity untranslated or was it only reencoded? BTW it moved into the DTD.

File: help/pt/Help_tips-appendix.xml [hunk 4]
```diff
@@ -25,7 +22,7 @@
 <appendix id="tips">
 	<appendixinfo>
 		<releaseinfo>
-			&app; Version 2.4.11 + Finance::Quote Version 1.18, April 2013
+			&app; Versão 2.4.11 + Finance::Quote Version 1.18, Abril 2013
```
[the remark this answers is not in the document]
Comment (fellen) [reply help/pt/Help_tips-appendix.xml:25]: @pmraps I just watched, in the table about single sources the entry about alphavantage is still missing. It is an important source. Currency etc. are relying on it.

Can you fix it?

**Pull request closed without merging**